edx-platform/learninghub-theme/lms/templates/instructor/edx_ace/enrollenrolled/email/body.html

```diff
@@ -1,95 +1,184 @@
-                <br />
-            </p>
-        </td>
-    </tr>
-
-
-
-</table>
+
+{% static 'images/PearsonLogo_Horizontal_Blk_RGB_color.png' as logo_url%}
+{% static 'images/learninghub-email-banner.png' as banner_url%}
+
+<div style="
+    background-color: #efefef;
+    font-family: arial,helvetica,
+    sans-serif;font-size: 16px;
+    color: #808080;
+    line-height: 1;
+    padding: 20px;">
+
+    <div style="
+        font-size:1px;
+        display:none!important">
+        %%_PreHeader%%
+    </div>
+
+    <table align="center" style="
+        line-height:1;
+        padding: 10px;
+        width: 600px;
+        border-spacing: 7px;">
+        <tbody>
+            <tr>
+                <td style="padding:0px">
+                    <div style="text-align:right">
+                        <span style="font-size:12px">
+                            <span style="font-family:arial,helvetica,sans-serif">
+                                <a style="color:#333333">View online</a>
+                            </span>
+                        </span>
+                    </div>
+                </td>
+            </tr>
+
+            <tr>
+                <td align="left">
+                    <img src="{% with_link_tracking logo_url %}" style="
+                    height: 61px;
+                    display: block;
+                    padding: 0;
+                    text-align: left">
+                </td>
+            </tr>
+
+            <tr>
+                <td align="center">
+                    <a href="{{ site_name }}" title="Pearson LearningHub" target="_blank" data-saferedirecturl="{{ site_name }}">
+                        <img src="{% with_link_tracking banner_url %}" alt="Pearson LearningHub" style="
+                            display:block;
+                            height:auto;
+                            width:600px;
+                            padding:0px;
+                            text-align:center;">
+                    </a>
+                </td>
+            </tr>
+
+            <tr>
+                <td style="
+                    padding:20px 20px 10px;
+                    background-color: white;">
+                    <span style="font-size:14px">
+                        <span style="font-family:Arial,Helvetica,sans-serif">
+
+                            <span style="color:#333333">
+                                {% if full_name%}
+                                    Dear {{ full_name }},<br><br>
+                                {% else %}
+                                    Dear {{ email_address }},<br><br>
+                                {% endif %}
+                                You have been enrolled in {{ display_name }} at&nbsp;
+                            </span>
+
+                            <a href="{{ site_name }}" style="color:#007fa3;text-decoration:underline" title="{{ site_name }}" target="_blank" data-saferedirecturl="{{ site_name }}">
```

Comment:
Please, add a space after the colon. @andrey-canon
Split the attributes on their own line to avoid line too long.